docs/developer-access-policy.md

```diff
@@ -0,0 +1,100 @@
+# Policy for OpenSAFELY Access by Platform Developers
+
+## Introduction
+
+OpenSAFELY securely mediates access to pseudonymized patient data for the purpose of analysis (be that research, service evaluation or audit).
+Our objective is to minimise direct access to pseudonymised data wherever possible, to protect privacy;
+while facilitating high quality, efficient analytics on the data by a wide range of users.
+A small number of OpenSAFELY team members need more direct access to the raw pseudonymised data inside the secure environment
```

Comment:
Nit: We could remove "raw" here, IMHO. We don't use it in the first sentence (above) and, thinking defensively, "raw" might seem to contradict "pseudonymized".